Dockerfile.design

```diff
@@ -0,0 +1,21 @@
+FROM decidim/decidim:latest
+LABEL maintainer="info@codegram.com"
+
+ENV LANG C.UTF-8
+ENV LC_ALL C.UTF-8
+ENV RAILS_ENV production
+ENV PORT 3000
+ENV SECRET_KEY_BASE=no_need_for_such_secrecy
+ENV RAILS_SERVE_STATIC_FILES=true
+
+WORKDIR /code
+COPY . .
+RUN bundle install
+RUN bundle exec rake install_all
```

Comment:
I don't think you need this since the application's Gemfile references the local version of decidim by path.

Reply:
Definitely, it was just me experimenting. Removing this!

Reply:
:+1: